autospec/config.py

```diff
@@ -551,6 +551,28 @@ def read_conf_file(self, path, track=True):
         lines = self.read_file(path, track=track)
         return [l.strip() for l in lines if not l.strip().startswith("#") and l.split()]
 
+    def process_extras_file(self, fname, name, filemanager):
+        """Process extras type subpackages configuration."""
+        content = {}
+        content['files'] = self.read_conf_file(os.path.join(self.download_path, fname))
+        if not content:
```

Comment:
I see that this code lived elsewhere before, but will this condition ever be true? `content['files']` will be populated whether or not there is an error (in which case it is `[]`).

Reply:
Yea, this condition should be fixed. I'll capture that in another PR.